.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
`.vscode` was already added to `.gitignore` but perhaps too late.

Do:
`git rm --cached -r .vscode`
then commit to remove it

Reply:
ooh TIL

Reply:
done, thanks guys!